recipes/aiokafka/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "aiokafka" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 36a5aa5bd681455c59c0de8a0c1bb9cfd2b32d93481bf56e821e9bad24859415
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.4
+    - pip
+    - nodejs
```

Comment:
Is `nodejs` actually needed? A quick glance at the `setup.py` wasn't clear.

Also, shouldn't `cython` be under `host` and then you need a c-compiler under `build`?
https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L38-L41

I don't see where it is used as a run time dependency. Building the cython c-extensions does look to be optional though, so you should decide if you want to distribute a binary that has them built that way. Also if you do, you'll need to drop the `noarch: python`.